docs/releasenotes/release-1.125.md

```diff
@@ -1,53 +1,43 @@
-* `PrivilegedAccessManagerEntitlement` is now a v1beta1 resource.
-* `RedisCluster` is now a v1beta1 resource.
-* `WorkstationCluster` is now a v1beta1 resource.
-
-## Modified Beta Reconciliation
+## New Fields:
 
-We migrated the following reconciliation from the TF-based or DCL-based controller to the new Direct controller to enhance the reliability and performance. The resource CRD is unchanged.
+* `AlloyDBInstance`
+  * Added `spec.networkConfig.enableOutboundPublicIp` field.
+  * Added `status.outboundPublicIpAddresses` field.
 
-* `ComputeFirewallPolicyRule`
+## Modified Beta Reconciliation
 
-  * You can use the alpha.cnrm.cloud.google.com/reconciler: direct annotation on ComputeFirewallPolicyRule resource to opt-in
-    the Direct Cloud Reconciler, which fixes the issue when updating `targetResources`.
+We migrated the following resources from the Terraform-based or DCL-based controller to the new Direct Controller to enhance the reliability and performance. The resource CRD is unchanged.
 
 * `SQLInstance`
 
-  * You can use the alpha.cnrm.cloud.google.com/reconciler: direct annotation on SQLInstance resources to opt-in
-    the Direct Cloud Reconciler, which fixes issues with updating from ENTERPRISE -> ENTERPRISE_PLUS edition and allows
-    "create from clone" functionality.
-
-## New Resources:
+  * You can use the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation on the `SQLInstance` CR object to opt-in the direct controller, which fixes issues with updating from ENTERPRISE -> ENTERPRISE_PLUS edition and allows "create from clone" functionality.
 
-* Added support for `PlaceholderKind` (v1beta1) resource.
-
-## New Fields:
+* `ComputeFirewallPolicyRule`
 
-* `AlloyDBInstance`
-  * Added `spec.networkConfig.enableOutboundPublicIp` field.
-  * Added `status.outboundPublicIpAddresses` field.
+  * You can use the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation on the `ComputeFirewallPolicyRule` CR object to opt-in the direct controller, which fixes the `targetResources` error *required value "priority" could not be found*.
 
 ## New features:
 
-* Allow more customization of resource reconciliation in cluster mode
-  * Added a new `ControllerReconciler` CRD (v1alpha1). See [example](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/operator/config/samples/controller_reconciler_customization_sample.yaml)
-  * This feature allows users to customize the client-side kube-apiserver request rate limit when Config Connector is runnning in cluster mode.
+* Add *cluster mode* to manage the rate-limit for the Config Connector requests
 
-## Bug Fixes:
+  * Previously we added [rate-limit control in namespace mode](https://cloud.google.com/config-connector/docs/how-to/customize-controller-manager-rate-limit). Users can configure the `ControllerReconciler` object to set the rate-limit for the reconciling requests to the kube-apiserver for their Config Connector resources. 
```

Comment:
The previously added CRD was `NamespacedControllerReconciler`, which is for namespaced mode. We added `ControllerReconciler` in this release for the cluster mode.

Reply:
Good call! Fixed.